keylime/agentstates.py

```diff
@@ -186,4 +233,4 @@ def add(self, agent_id, boottime, ima_pcrs_dict, next_ima_ml_entry, learned_ima_
         agentAttestState.set_boottime(boottime)
         agentAttestState.set_ima_pcrs(ima_pcrs_dict)
         agentAttestState.set_next_ima_ml_entry(next_ima_ml_entry)
-        agentAttestState.set_ima_keyrings(ImaKeyrings.from_json(learned_ima_keyrings))
+        agentAttestState.set_ima_keyrings(ImaKeyrings.from_dict(learned_ima_keyrings))
```

Comment:
Here you accidentally changed it for the ImaKeyrings class

Reply:
ooops, poor attempt at multi-tasking, fixed now